src/Components/Notifications/util.ts

```diff
@@ -14,3 +17,28 @@ const isArtworksBasedNotification = (
 ) => {
   return ["ARTWORK_ALERT", "ARTWORK_PUBLISHED"].includes(notificationType)
 }
+
+export const setLastSeenNotificationDate = date => {
```

Comment:
Why extract this? Just inline this in the `useEffect` so it isn't misused. If it is extracted however you should annotate the type.